source/community/reactnative/src/visualizer/renderer.js

```diff
@@ -169,7 +181,7 @@ export default class Renderer extends React.Component {
                 ],
                 { cancelable: false }
             )
-        } else if (actionObject.type === "Action.OpenUrl" && !Utils.isNullOrEmpty(actionObject.url)) {
+        } else if (actionObject.type === "Action.OpenUrl" && !Utils.isNullOrEmpty(actionObject.url) || actionObject.type === "CustomActionButton") {
```

Comment:
`CustomActionButton` can be anything such as `URL / submit or showCard`. we can just alert it with the payload details. Please use **_`alertAction`_** method. I think we are already calling **_`alertAction`_** by default if it's not `Submit/URL`. So you can remove this condition.

Reply:
Resolved